src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/ResettlementAssessmentController.kt

```diff
@@ -313,4 +314,47 @@ class ResettlementAssessmentController(
     @Parameter(required = true)
     pathway: Pathway,
   ) = resettlementAssessmentService.getLatestResettlementAssessmentByNomsIdAndPathway(nomsId, pathway, resettlementAssessmentStrategies)
+
+  @PostMapping("/{nomsId}/resettlement-assessment/skip", produces = [MediaType.APPLICATION_JSON_VALUE])
+  @Operation(summary = "Skip an assessment")
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "204",
```

Comment:
Usually it's 202, but if the rest of the project is 204 I guess better being consistent